src/droplet.F

```diff
@@ -519,16 +516,7 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
 
     if(timestats.ge.1) time_phys_D2H=time_phys_D2H+mytime()
 
-    numvec(1) = num100
-    numvec(2) = num1000
 
-#ifdef MPI
-    if ( myid == 0 ) then
-       call mpi_reduce(mpi_in_place,numvec,2,mpi_int,mpi_sum,0,mpi_comm_world,ierr)
-    else
-       call mpi_reduce(numvec,numvec,2,mpi_int,mpi_sum,0,mpi_comm_world,ierr)
-    end if
-#endif
```

Comment:
So the update of `num100` and `num1000` is moved to `droplet_diag` now?

Reply:
That's right. There is no longer any point doing it here. They are now part of the droplet_diag0 array